EDAM_dev.owl

```diff
@@ -21037,10 +21037,12 @@
         <rdfs:subClassOf rdf:resource="http://edamontology.org/data_2091"/>
         <rdfs:subClassOf rdf:resource="http://edamontology.org/data_2900"/>
         <created_in>beta12orEarlier</created_in>
-        <oboInOwl:hasDefinition>Identifier of an entry from the GlycosciencesDB database.</oboInOwl:hasDefinition>
+        <oboInOwl:hasDefinition>The Linucs ID is used to identify carbohydrate sequences using a linear and canonical description.</oboInOwl:hasDefinition>
```

Comment:
We need to investigate if there is an actual Linucs ID or if it is a notation. If so this conept would need to be a format and not a data.

Reply:
Turns out Linucs notation and Linus ID are 2 different things.

Reply:
Million thanks for diving into this and finding it all out🙏🏽

I'm suggesting this further TODO with the LINUCS stuff:
- [ ] Add regex for the ID, which is a natural number starting from `1` (at the moment they are max 5 digits, but that may change at some point(how soon?))
- [ ] Add format for the LINUCS notation (under something like Carbohydrate (sequence/Glycan) format?)
- [ ] Add the citation `https://doi.org/10.1016/S0008-6215(01)00230-0` in the format, remove from the LINUCS ID. Perhaps the same with the fully spelled-out synonym.
- [ ] Maybe a comment in the ID, noting the fact about the LINUCS notation
- [ ] I'd suggest label `LINUCS ID` and exact synonym `linucsid` like in the DB
- [ ] Revise all the "Carbohydrate" & "Glyc.." concepts in EDAM (incl. Glycoinformatics😊). For example there is "Carbohydrate structure" which is defined only as 3D structure (not in 2021😉). GlycosciencesDB has 4 formats for "2D" structure: CarbBank LINUCS GlycoCT_condensed GlycoCT_xml. Maybe any of the visualisations shoudl be added? (Not sure)
- [ ] Part of the revision: "Protein" 3D structure formats are usually generic for any molecules, incl. glycans. Revise accordingly.
@LucieLamothe I'd be cool to have a chat about this a decide what we can do now, and what should wait until 2022. Million thanks again🙇🏽‍♂️